scripts/ci/github/verify_updated_weights.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
One thing that comes to my mind is that we probably also want this in cumulus, so could be that the scripts should rather live in https://github.com/paritytech/pipeline-scripts/

Although on the other hand we should "soon" have our monorepo, in which case it needs to be shuffled around anyway.